charts/scalardb/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: scalardb
+description: A Helm chart of Scalar DB
+
+type: application
+version: 1.0.0
+appVersion: 3.1.0
+deprecated: false
+icon: https://scalar-labs.com/wp-content/themes/scalar/assets/img/logo_scalar.svg
+keywords:
+- scalardb
+- scalardb-server
```

Comment:
Is it better to add `grpc` or `grpc-server` to the keywords?

Reply:
Thanks for the suggestion. I'll add them.